test/cfg/std.c

```diff
@@ -219,14 +219,12 @@ void bufferAccessOutOfBounds(void)
     strncpy_s(a,5,"abcd",5);
     // string will be truncated, error is returned, but no buffer overflow
     strncpy_s(a,5,"abcde",6);
-    // TODO cppcheck-suppress bufferAccessOutOfBounds
-    strncpy_s(a,5,"a",6);
     strncpy_s(a,5,"abcdefgh",4);
     // valid call
     strncat_s(a,5,"1",2);
     // cppcheck-suppress bufferAccessOutOfBounds
     strncat_s(a,10,"1",2);
-    // TODO cppcheck-suppress bufferAccessOutOfBounds
+    // cppcheck-suppress bufferAccessOutOfBounds
```

Comment:
I have the feeling this is also a false positive. I have the feeling that

    strncat_s(a,5,"1",2);

and 

    strncat_s(a,5,"1",5);

are logically the same.